README.md

```diff
@@ -14,14 +14,15 @@ package main
 import(
     "fmt"
     "crypto/sha256"
-    "github.com/lazyledger/smt"
+    "github.com/celestiaorg/smt"
 )
 
 func main() {
-    // Initialise a new key-value store to store the nodes of the tree
-    store := smt.NewSimpleMap()
+    // Initialise two new key-value store to store the nodes and values of the tree
+    nodestore := smt.NewSimpleMap()
+    valuestore := smt.NewSimpleMap()
```

Comment:
I think these should be camel cased according to the naming convention.

Reply:
Fixed in 738728c